service/dap/server.go

```diff
@@ -567,13 +564,13 @@ func (s *Server) onConfigurationDoneRequest(request *dap.ConfigurationDoneReques
 	}
 	s.send(&dap.ConfigurationDoneResponse{Response: *newResponse(request.Request)})
 	if !s.args.stopOnEntry {
-		s.doContinue()
+		s.doCommand(api.Continue, "breakpoint")
 	}
 }
 
 func (s *Server) onContinueRequest(request *dap.ContinueRequest) {
 	s.send(&dap.ContinueResponse{Response: *newResponse(request.Request)})
-	s.doContinue()
+	s.doCommand(api.Continue, "breakpoint")
```

Comment:
Will delve dap never use `allThreadsContinued` in the response?

Reply:
Good eye! This should be set now. This code works as needed - the value is omitted and is assumed to be true by the client, but only because we have not updated the dependency on go-dap after a but fix there to support false values (see https://github.com/google/go-dap/pull/40). I will update this in a separate PR.